components/VFBMain.js

```diff
@@ -576,10 +571,7 @@ export default class VFBMain extends React.Component {
 
   renderHTMLViewer (htmlChild) {
     if (htmlChild !== undefined) {
-      this.setState({
-        UIUpdated: true,
```

Comment:
this should make it back since it's used for the ShouldComponentUpdate to minimise the not needed re-render